warp-core/src/test/scala/com/workday/warp/persistence/mysql/WarpSlickDslSpec.scala

```diff
@@ -50,7 +50,7 @@ class WarpSlickDslSpec extends WarpJUnitSpec with CorePersistenceAware {
 
   /** Tests QUOTE dsl. */
   @UnitTest
-  def returnQuoted(): Unit = {
+  def returnQuoted(): Unit = skipIfH2 {
```

Comment:
I'm a little lost at how/why this works could you explain a little more?

Reply:
This is from Tomas' code, but here's the `skipIfH2` function at `SkipIfH2.scala:12`
```scala
  def skipIfH2[T](f: => T): Option[T] = {
    this.profile match {
      case H2Profile => None
      case _ => Option(f)
    }
  }
```
So it's a function that takes a function `f` and returns an `Option`. We can wrap `returnQuoted()`'s body in this call to `skipIfH2` which will simply return `None` instead of running the given test body; the resulting `Option` type doesn't matter since the unit test's returning `Unit`

A quick repl mockup using a boolean parameter instead of matching on the profile
```scala
scala> def dayman[T](isTuesday: Boolean, f: => T): Option[T] = {
     |   isTuesday match {
     |     case false => None
     |     case true => Option(f)
     |   }
     | }
dayman: [T](isTuesday: Boolean, f: => T)Option[T]

scala> def x(i: Int): Unit = print(i)
x: (i: Int)Unit

scala> def someTest(): Unit = dayman(true, x(4))
someTest: ()Unit

scala> someTest()
4
scala> def someTest2(): Unit = dayman(false, x(4))
someTest2: ()Unit

scala> someTest2()
[edit: just a blank line, no result printed to the output]
```